src/server/api/inbox.ts

```diff
@@ -113,6 +113,31 @@ export const inboxRoutes = (cfg: APIConfig, store: Store, apsystem: ActivityPubS
     return await reply.send({ message: 'ok' })
   })
 
+  // TODO: Paging?
+  server.get<{
+    Params: {
+      actor: string
+      inReplyTo: string
+    }
+    Reply: APOrderedCollection | string
+  }>('/:actor/outbox/replies/:inReplyTo', {
+    schema: {
+      params: Type.Object({
+        actor: Type.String(),
+        inReplyTo: Type.String()
+      }),
+      description: 'Replies for a post',
+      tags: ['ActivityPub']
+    }
+  }, async (request, reply) => {
+    const { actor, inReplyTo } = request.params
+
+    // No need for verification?
```

Comment:
i think we need to distinguish audiences, a public post may have private replies and we don't want to attach those to the collection for unauthed requests.

unless, as i say below about referencing rather than storing replies' objects, we just give a list of known replies, but the remote instance decides if an unauthed client can fetch it, including denying access to private replies. 

though publicly announcing there's a reply that may be private, specially since most ids contain the username, is iffy to me. can we index objects by audiences too?

Reply:
Good point, I think instead of another index we can filter it during iteration.

Reply:
so only show replies addressed to `https://www.w3.org/ns/activitystreams#Public` on public fetches and everything else for a request signed by the actor, right?

Reply:
Added a filter and attempt to fetch the `to` field if the request is signed, ignore otherwise and only show public posts.